requirements.txt

```diff
@@ -1,4 +1,4 @@
 boto3>=1.10.0
-cryptography>=3.4.0
+cryptography>=3.4.6
```

Comment:
Because I don't have it to hand: What version is the one that starts to require Rust?

Reply:
This change is already on master: https://github.com/aws/aws-encryption-sdk-python/blob/master/requirements.txt
I'll do some git to fix this diff

Reply:
Done, needed to merge master into `mpl-reviewed` feature branch